Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -479,6 +484,8 @@ void WebFrame::commitProvisionalFrame()
 
     if (ownerElement)
         ownerElement->scheduleInvalidateStyleAndLayerComposition();
+
+    localFrame->setIsProvisional(false);
```

Comment:
It would be less error prone if this were right next to each use of m_provisionalFrame in this file.  Then we would know we got them all and the transition time would be a little more accurate in case there's something in the same function call that uses the bool.